streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -74,10 +74,12 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
     directory: String,
     filter: Path => Boolean = FileInputDStream.defaultFilter,
     newFilesOnly: Boolean = true,
-    conf: Option[Configuration] = None)
+    @transient conf: Option[Configuration] = None)
```

Comment:
Of course, but you've changed it to use `serializableConfOpt` now. `conf` need no longer even be a field, right? in fact you can't use it anymore since it will be `null` after deserialization; it's not restored.